crunchy.go

```diff
@@ -158,6 +161,15 @@ func (v *Validator) Check(password string) error {
 	if countUniqueChars(password) < v.options.MinDiff {
 		return ErrTooFewChars
 	}
+	idx, err := foundInHIBP(password)
```

Comment:
We should first check for `v.options.CheckHIBP` being true, before calling `foundInHIBP`. Otherwise we're still always making the API call, we're just ignoring its results.

Reply:
you are right, fixed it. wrote it bad due to rewriting